### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by szepeviktor**
Found few misspellings.

Notebooks are excluded.
Willing contributors [may work](https://github.com/crate-ci/typos) on those.

### CODE REVIEW

**Review by younesbelkada — APPROVED** [review]
Thanks for fixing the typos!

**Comment by szepeviktor** [comment]
Glad to contribute.

File: src/peft/tuners/lora/tp_layer.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def __init__(
         LoraLayer.__init__(self, base_layer=base_layer)
 
         self.backend = backend
-        self.is_paralle_a = isinstance(base_layer, backend.RowParallelLinear)
+        self.is_parallel_a = isinstance(base_layer, backend.RowParallelLinear)
```
Comment (younesbelkada) [inline src/peft/tuners/lora/tp_layer.py:54]: Actually this is a breaking change - @szepeviktor would you mind opening a PR to re-add `self.is_paralle_a` as a property method with a FutureWarning for backward compatibility ?

**Comment by younesbelkada** [comment]
Just made https://github.com/huggingface/peft/pull/1439 to fix the breaking change

**Pull request merged**